src/app/app.component.spec.ts

```diff
@@ -14,19 +14,19 @@ describe('AppComponent', () => {
     }).compileComponents();
   }));
 
-  it('should create the app', () => {
```

Comment:
Any reason why you replaced 'it' with 'xit'? The test runner excludes these when you run the unit tests. If you feel some of them are not required at all, remove them from the spec file instead.

Reply:
I will be removing them since they are not required in this functionality at all.